sql/src/test/java/io/crate/planner/SelectPlannerTest.java

```diff
@@ -704,4 +705,12 @@ public void testFilterOnPKSubsetResultsInPKLookupPlanIfTheOtherPKPartIsGenerated
             "Get[doc.t_pk_part_generated | 1 | DocKeys{0, 0}"
         ));
     }
+
+    @Test
+    public void testInnerEquiJoinWithHashJoin() throws Exception {
```

Comment:
maybe rename to clarify the test scenario - e.g. `testInnerJoinResultsInHashJoinIfHashJoinIsEnabled`